habitat/config/default.py

```diff
@@ -17,8 +18,10 @@ def __init__(self, *args, **kwargs):
 
 CN = Config
 
-DEFAULT_CONFIG_DIR = "configs/"
 CONFIG_FILE_SEPARATOR = ","
+_HABITAT_ROOT = osp.dirname(osp.dirname(osp.abspath(__file__)))
```

Comment:
This will not work in ipython or colab.

Reply:
Changed this to `inspect.getabsfile(inspect.currentframe())` as that works for everything

Reply:
Oh nice!